docs/html/reference/pip_download.rst

```diff
@@ -69,51 +90,118 @@ Examples
     (or more generic, i.e. ``2``) and implementation to ``cp`` (or more generic, i.e. ``py``).
 
-    ::
+    .. tabs::
 
-      $ pip download \
+      .. group-tab:: Unix/macOS
+
+        .. code-block:: shell
+
+          python -m pip download \
           --only-binary=:all: \
           --platform macosx-10_10_x86_64 \
           --python-version 27 \
           --implementation cp \
           SomePackage
 
+      .. group-tab:: Windows
+
+        .. code-block:: shell
+
+          py -m pip download ^
+          --only-binary=:all: ^
+          --platform macosx-10_10_x86_64 ^
+          --python-version 27 ^
+          --implementation cp ^
+          SomePackage
+
 #. Download a package and its dependencies with linux specific constraints.
     Force the interpreter to be any minor version of py3k, and only accept
     ``cp34m`` or ``none`` as the abi.
 
-    ::
+    .. tabs::
+
+      .. group-tab:: Unix/macOS
+
+        .. code-block:: shell
 
-      $ pip download \
+          python -m pip download \
           --only-binary=:all: \
           --platform linux_x86_64 \
           --python-version 3 \
           --implementation cp \
           --abi cp34m \
           SomePackage
 
+      .. group-tab:: Windows
+
+        .. code-block:: shell
+
+          py -m pip download ^
+          --only-binary=:all: ^
+          --platform linux_x86_64 ^
+          --python-version 3 ^
+          --implementation cp ^
+          --abi cp34m ^
+          SomePackage
+
 #. Force platform, implementation, and abi agnostic deps.
 
-    ::
+    .. tabs::
+
+      .. group-tab:: Unix/macOS
 
-      $ pip download \
+        .. code-block:: shell
+
+          python -m pip download \
           --only-binary=:all: \
           --platform any \
           --python-version 3 \
           --implementation py \
           --abi none \
           SomePackage
 
+      .. group-tab:: Windows
+
+        .. code-block:: shell
+
+          py -m pip download ^
+          --only-binary=:all: ^
+          --platform any ^
+          --python-version 3 ^
+          --implementation py ^
+          --abi none ^
+          SomePackage
+
 #. Even when overconstrained, this will still correctly fetch the pip universal wheel.
 
-    ::
+    .. tabs::
 
-      $ pip download \
-          --only-binary=:all: \
-          --platform linux_x86_64 \
-          --python-version 33 \
-          --implementation cp \
-          --abi cp34m \
-          pip>=8
-      $ ls pip-8.1.1-py2.py3-none-any.whl
-      pip-8.1.1-py2.py3-none-any.whl
+      .. group-tab:: Unix/macOS
+
+        .. code-block:: console
+
+          $ python -m pip download \
+            --only-binary=:all: \
+            --platform linux_x86_64 \
+            --python-version 33 \
+            --implementation cp \
+            --abi cp34m \
+            pip>=8
+
+          $ ls pip-8.1.1-py2.py3-none-any.whl
+          pip-8.1.1-py2.py3-none-any.whl
```

Comment:
I'm not happy about this, but don't have any suggestion other than giving `ls` another code-block.